src/PerfView/PerfViewData.cs

```diff
@@ -5713,6 +5713,10 @@ protected internal override StackSource OpenStackSourceImpl(string streamName, T
                 var heapParser = new HeapTraceProviderTraceEventParser(eventSource);
                 Dictionary<Address, StackSourceSample> lastHeapAllocs = null;
 
+                Dictionary<TraceModuleFile, NativeSymbolModule> loadedModules = new Dictionary<TraceModuleFile, NativeSymbolModule>();
+                var allocationTypeNames = new Dictionary<CallStackIndex, string>();
+                var symReader = GetSymbolReader(log);
```

Comment:
Is this the appropriate way to get a SymbolReader? The CacheOnly argument can skip pdbs, which results in not finding any typenames for allocations from those modules. Even if I later load symbols for a module, I don't see a way to refresh the entire stackSource.